package.json

```diff
@@ -80,8 +80,8 @@
     "plop": "^4.0.0",
     "prettier": "^3.5.3",
     "prop-types": "^15.8.1",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
+    "react": "19.1.0",
+    "react-dom": "19.1.0",
```

Comment:
Could you test if this breaks things? I don't think it should, would be nice to have wider react version compatibility than just 19.1.X

```suggestion
    "react": "^19.1.0",
    "react-dom": "^19.1.0",
```